src/Middleware/RequestThrottling/src/QueuePolicies/QueuePolicyOptions.cs

```diff
@@ -5,9 +5,9 @@
 namespace Microsoft.AspNetCore.RequestThrottling.Policies
 {
     /// <summary>
-    /// Specifies options for the <see cref="TailDrop"/>
+    /// Specifies options for the <see cref="TailDropQueuePolicy"/>
```

Comment:
Since this option is for any kind of `IQueuePolicy`, should this cref change?
```suggestion
    /// Specifies options for the <see cref="IQueuePolicy"/>
```

Reply:
good catch, fixed